### DISCUSSION TITLE
Read from Unity catalog

### DISCUSSION THREAD

**Discussion by royb131** [delta-io/delta-rs#2450] [Q&A]
All, trying to read unity catalog table using python, anyone seeing the issue.
`DeltaTable.from_data_catalog(data_catalog=data_catalog, data_catalog_id=catalog_name, database_name=schema_name, table_name=table_name)`
`AttributeError: type object 'deltalake._internal.RawDeltaTable' has no attribute 'get_table_uri_from_data_catalog'`

Same for glue as well. Thanks

**Comment by christophergrant** [comment]
@royb131 how did you resolve this?
